pkg/controller/utils/clusterinfo.go

```diff
@@ -61,6 +61,9 @@ func GetMasterNodeSubnets(kclient client.Client) (map[string]string, error) {
 	if err != nil {
 		return subnets, err
 	}
+	if len(machineList.Items) == 0 {
+		return subnets, fmt.Errorf("Did not find any master Machine objects")
+	}
```

Comment:
nts: this appears to be the only non-test change -- I guess a bug uncovered by the new tests.

Reply:
Yes, in one of the test cases I forgot to add `Machine` objects (a case which should never occur in OSD) and so I thought it best to handle in the `utils` package. When there are no `Machine` objects returned by the List method, there is a subscript reference as a few lines down the first object is referenced by `machineList.Items[0]`.